lib/netext/dialer.go

```diff
@@ -265,3 +321,7 @@ func (c *Conn) Write(b []byte) (int, error) {
 	}
 	return n, err
 }
+
+func init() {
+	rand.Seed(time.Now().UTC().UnixNano())
+}
```

Comment:
I would prefer if the resolver has a random generator, and given that it needs to be concurrently called (http.batch) that will be a little more code :( 
So maybe in the next PR

Reply:
Why would it be a problem with `http.batch()`? I made it internal to `resolver` in f80540fa, and I haven't seen any panics. Let me know if we need something else.